Doc/library/functools.rst

```diff
@@ -358,6 +378,31 @@ The :mod:`functools` module defines the following functions:
       >>> basetwo('10010')
       18
 
+   If :data:`Placeholder` sentinels are present in *args*, they will be filled first
+   when :func:`partial` is called. This allows custom selection of positional arguments
+   to be pre-filled when constructing :ref:`partial object<partial-objects>`.
+   If :data:`Placeholder` sentinels are used, all of them must be filled at call time:
```

Comment:
We try to keep it to one unique ref pr. paragraph, in order to reduce the visual clutter.

```suggestion
   If :data:`!Placeholder` sentinels are used, all of them must be filled at call time:
```